pylsp/_utils.py

```diff
@@ -10,6 +10,8 @@
 import threading
 from typing import List, Optional
 
+import time
+from functools import wraps
```

Comment:
Please move these two imports to the standard library section above.

Reply:
I'm wondering: can't we automate that somehow? :)

Reply:
`isort` maybe, but I'm not really sure.